inspirehep/modules/editor/views.py

```diff
@@ -54,3 +56,18 @@ def preview():
     endpoint = get_endpoint_from_record(record)
     template = current_app.config['RECORDS_UI_ENDPOINTS'][endpoint]['template']
     return render_template(template, record=record)
+
+
+@blueprint.route('/holdingpen/<int:workflow_id>', methods=['GET'])
+@login_required
```

Comment:
Maybe just for superusers and curators? That means anyone logged can retrieve info about this workflow